app/scripts/controllers/paymentModal.js

```diff
@@ -39,6 +39,13 @@ angular.module('confRegistrationWebApp')
         alert('Transaction amount must be a positive number.');
         return;
       }
+      if ($scope.newTransaction.paymentType === 'CREDIT_CARD') {
+        var validationError = ccp.validateCardNumber($scope.newTransaction.creditCard.number || '');
+        if(validationError){
+          alert('Please enter a valid card number. The ' + validationError + '.');
+          return;
+        }
```

Comment:
That's not quite what I meant. It's clearly using ccp.js here. But you also use it payment.js. Why do you need to do both?